README.md

```diff
@@ -1,2 +1,81 @@
-# kanban-board
-JavaScript Grop
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: JavaScript capstone project - Food Palace
+
+## Description the project.
+> This JavaScript capstone project was based on building our own web application based on an external API.
+
+## Below are some of the task we implemented based on the project requirement;
+- [x] A home page showing a list of items that you can "like."
+- [x] A popup window with more data about an item that you can use to comment on it or reserve it   for a period of time.
+
+
+## Built With
+
+### Major languages
+- [x] HTML
+- [x] CSS
+- [x] JavaScript
+- [x] Bundle Webpack
+
+### Frameworks
+- None
+### Technologies used
+- None
+## Live Demo (if available)
+
+## [Live Demo Link](https://doheera-kosi.github.io/Food-Palace/dist/)
+
+## [Recorded Video](https://drive.google.com/file/d/1cV0b2Cw-4z50xmHE8tuCtcjFJZ-775b2/view?usp=sharing)
+
+
+## Getting Started
+
+### To get a local copy up and running follow these simple example steps.
+
+- Clone the repo to your local environment
+- Run npm install to install all dependencies
+- Open the index.html file with your favorite browser.
+- Modify to your taste with your preferred text editor
+- Enjoy 🤗
+
+
+
```

Comment:
- [ ] Good job so far 👏 Your README looks fine but as your web app uses 2 APIs, It is essential to provide enough information about it. For the case of the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) you know someone needs to have the key. Therefore, you should state that also here in this section. In short, please try to provide information about the APIs that your app consumes and how to use them 😃 👍

Reply:
Hi, @BertrandConxy
Thank you for the feedback. We have added the APIs links for the Project!